sktime/utils/validation/series.py

```diff
@@ -69,10 +70,12 @@ def check_series(
     if enforce_univariate:
         _check_is_univariate(Z)
 
-    # check time index
-    check_time_index(
-        Z.index, allow_empty=allow_empty, enforce_index_type=enforce_index_type
-    )
+    # check time index if input data is not an NumPy ndarray
+    if not isinstance(Z, np.ndarray):
```

Comment:
@mloning  and @fkiraly note that this function already had option to accept a ndarray. However, the call to check_time_index was using the `.index` attribute that a ndarray does not have. Just added guardian code to avoid this if a ndarray is being checked. 

Also added extra doc to make clearer that ndarrays were already in the allowed input type.